src/ir/cfg_builder.cpp

```
@@ -127,6 +128,121 @@ static bool has_fall(const Instruction& ins) {
+}
+
+static bool un_requires_base64(const Un& un) {
+    switch (un.op) {
+    case Un::Op::BE64:
+    case Un::Op::LE64:
+    case Un::Op::SWAP64: return true;
+    default: return false;
+    }
+}
+
+static std::optional<RejectionReason> check_instruction_feature_support(const Instruction& ins,
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)